nlp_models/tests/test_nlp_article.py

```diff
@@ -0,0 +1,11 @@
+import unittest
```

Comment:
Need more working tests, not comments, create only few simple tests for now and check with db